governance/program/src/state/legacy.rs

```diff
@@ -0,0 +1,256 @@
+//! Legacy Accounts
```

Comment:
Yes, I copied them. I initially referenced the v1 crate to get the structs as they were but that caused issues which I thought added overheard not worth paying for.

- The v1-v2 mapping code become unnecessarily complex because each enum/struct used for fields had to be mapped even though they didn't change between versions 
- `VSCode` kept resolving/offering the v1 versions of the structs which was annoying and also very easily to accept and get it accidentally wrong.